README.md

```diff
@@ -96,7 +96,7 @@ installation of the following files:
     sudo cp ratbagd.service /etc/systemd/system/ratbagd.service
 
 The files are installed into the prefix by make install, see also the
```

Comment:
Nitpicking, this is not `make install` anymore, but `ninja install`

Other than that looks good (feel free to force push an amended commit to replace this one, and I'll merge the PR).